### PR TITLE
Revert Pending Block support

### PR DESCRIPTION
**Pull request by hqjang-pepper**
## Proposed changes

- "pending keywords were disabled in klaytn APIs at https://github.com/klaytn/klaytn/pull/738 
- enable "pending" keywords again since this PR.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: governance/api.go [hunk 1]
```diff
@@ -263,36 +254,35 @@ func (api *GovernanceKlayAPI) GasPriceAtNumber(num int64) (uint64, error) {
...
-//
-// func (api *GovernanceKlayAPI) setDefaultTxGasHumanReadable() (uint64, error) {
-// 	err := api.governance.currentSet.SetValue(params.ConstTxGasHumanReadable, params.TxGasHumanReadable)
-// 	if err != nil {
-// 		return 0, errSetDefaultFailure
-// 	} else {
-// 		return params.TxGasHumanReadable, nil
-// 	}
-// }
+func (api *GovernanceKlayAPI) GetTxGasHumanReadable(num *rpc.BlockNumber) (uint64, error) {
```
Comment (yoomee1313) [inline governance/api.go:257]: I think those apis are disabled.. Can I ask why you enable those apis again?

File: api/api_public_blockchain.go [hunk 2]
```diff
@@ -164,10 +164,17 @@ func (s *PublicBlockChainAPI) GetHeaderByHash(ctx context.Context, hash common.H
 // transactions in the block are returned in full detail, otherwise only the transaction hash is returned.
 func (s *PublicBlockChainAPI) GetBlockByNumber(ctx context.Context, blockNr rpc.BlockNumber, fullTx bool) (map[string]interface{}, error) {
 	block, err := s.b.BlockByNumber(ctx, blockNr)
-	if err != nil {
-		return nil, err
+	if block != nil && err == nil {
```
Comment (yoomee1313) [inline api/api_public_blockchain.go:167]: @hqjang-pepper It seems when block and err are both nil, it returns nil, nil. Also, is it possible that block is nil?

File: governance/api.go (hunk 1 above)
Comment (hqjang-pepper) [reply governance/api.go]: I didn't realize that these apis are disabled regardless of pending blocks. Disabled again.

File: api/api_public_blockchain.go (hunk 2 above)
Comment (hqjang-pepper) [reply api/api_public_blockchain.go:167]: Since there is no case of return nil,nil in BlockByNumber at api_backend.go.
So return nil, nil is needless.

Comment (hqjang-pepper) [reply api/api_public_blockchain.go:167]: When block is nil, it always return err

**Review by yoomee1313 — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

File: api/api_public_blockchain.go (hunk 2 above)
Comment (jeongkyun-oh) [reply api/api_public_blockchain.go:167]: @hqjang-pepper According to https://github.com/klaytn/klaytn/pull/738/files, is it okay not to change `GetBlockByHash`?

**Review by jeongkyun-oh — APPROVED** [review]

File: api/api_public_blockchain.go (hunk 2 above)
Comment (hqjang-pepper) [reply api/api_public_blockchain.go:167]: @jeongkyun-oh Revision of `GetBlockByHash` is not related to pending block issue, so I think it's unnecessary.

**Pull request merged**